commit 6f05118b6e225d25ecb811eb6ae2f842687fb6fc
Author: Tom Schoonjans <Tom.Schoonjans@genomicsplc.com>
Date:   Thu Sep 1 10:15:14 2022 +0530

    Fix python-numpy-bindings meson bug
    
    This fix ensures that meson will error out when `python-numpy-bindings` is set to `enabled`, but `cython` is not found.

diff --git a/python/meson.build b/python/meson.build
index b3b4de2..9c12dd3 100644
--- a/python/meson.build
+++ b/python/meson.build
@@ -57,7 +57,7 @@ if not get_option('python-bindings').disabled()
 endif
 
 if not get_option('python-numpy-bindings').disabled()
-    cython = find_program('cython', 'cython3', 'cython-3', 'cython' + python.language_version(), 'cython-' + python.language_version(), required : false)
+    cython = find_program('cython', 'cython3', 'cython-3', 'cython' + python.language_version(), 'cython-' + python.language_version(), required : get_option('python-numpy-bindings'))
     deps = [python_dep, xraylib_lib_dep]
     # lld-link doesnt find the openmp import libraries
     if cc.get_id() != 'clang-cl'
